app/org/sagebionetworks/bridge/models/accounts/AccountId.java

```diff
@@ -68,12 +75,18 @@ public Phone getPhone() {
         }
         return phone;
     }
+    public String getHealthCode() {
+        if (usePreconditions && healthCode == null) {
+            throw new NullPointerException("AccountId.healthCode is null");
+        }
```

Comment:
Add healthCodeAccessorThrows() to AccountIdTest? Add a clause to getValuesWithoutGuards()?

Reply:
Updated with additional tests.